indico/htdocs/js/indico/jquery/colorswitch.js

```diff
@@ -0,0 +1,114 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    $.widget("indico.colorswitch", {
+
+        options: {
+            defaultColors: [
+                [
+                    {bgColor: '#EEE0EF', textColor: '#1D041F'},
```

Comment:
OTOH. If we define the colors in a scss file, how could we protect against requests sending invalid colors? Let's discuss.